README.md

```diff
@@ -1,3 +1,59 @@
 # Bangalore Clojure Group Website
 
-To be updated!
+[Visit the Website](https://bangalore-clj.github.io/)
+
+
+## About
+
+The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
```

Comment:
The meetups started much earlier than 2013 - check https://groups.google.com/forum/#!forum/bangalore-clj older posts for details.

Reply:
fixed